queries/slippage/subquery_batchwise_internal_transfers.sql

```diff
@@ -224,7 +234,8 @@ buffer_trades as (
                     and
                         -- We know that settlements - with at least one amm interaction - have internal buffer trades only if 
                         -- the solution must come from a internal_buffer_trader_solvers solver
-                        a.solver_address in (select * from internal_buffer_trader_solvers) 
+                        (a.solver_address in (select * from internal_buffer_trader_solvers) 
+                        or a.dex_swaps = 0)
```

Comment:
I still no not understand the meaning of this. Why is it not matching the criteria above?
```
(dex_swaps = 0 and num_trades < 2) or dex_swaps > 0
```